### ISSUE TITLE
Build fails on aws-eb-preconfigured docker amazon/aws-eb-python:3.4.2-onbuild-3.5.1

### ISSUE THREAD

**Issue by hornedbull** [dkarchmer/aws-eb-preconfigured-docker-django#1]
When fails while running this command with python not found. Any help?

container_commands:
  01migrate:
    command: "docker exec `docker ps -a --no-trunc -q | head -n 1` /var/app/bin/python /var/app/manage.py migrate --noinput  &>> /tmp/deploy.log"
    leader_only: true

**Comment by dkarchmer** [comment]
Are you running this locally? or on AWS? Did you change anything?

**Comment by dkarchmer** [comment]
I would recommend ssh-ing to the AWS EC2 machine, and try to see what you get with `docker ps -a --no-trunc -q | head -n 1` and then see if you can call migrate in some way with docker exec.
If you updated to use a newer AMI (instead of `64bit Debian jessie v1.3.1 running Python 3.4 (Preconfigured - Docker)`, that may be the issue, and you will have to figure out what has changed with the AMI or newer version of docker

**Comment by hornedbull** [comment]
Yes I’ve the newer AMI. Okay will share the outcome once I reach a conclusion. Thanks for helping out :)

Vatsal Shah
shahvatsal90@gmail.com
